code/Aspects/CacheAfterCallAspect.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace LeKoala\DebugBar\Aspect;
+
+use LeKoala\DebugBar\Collector\SilverStripeCollector;
+use Monolog\Logger;
+use SilverStripe\Core\Injector\AfterCallAspect;
+use SilverStripe\Core\Injector\Injector;
+
+class CacheAfterCallAspect implements AfterCallAspect
+{
+    /**
+     * Logs all hits/misses after a CacheInterface::get call is made.
+     *
+     * {@inheritdoc}
+     */
+    public function afterCall($proxied, $method, $args, $result)
+    {
+        $message = "Your partial cache named {$args[0]} ";
+        $message .= (empty($result)) ? 'didn’t find any existing caches' : 'found an existing cache';
+        SilverStripeCollector::addTemplateCacheInfo($message);
```

Comment:
@robbieaverill thanks for reviewing. I agree that SilverStripeCollector isn't ideal to store the array as it adds needless responsibility to the class. Instead this can be added to the Injected cache object as an array. 

Further, as discussed I'll make the results clickable so in the event of a cache having an arbitrary hash for a name, you can click on it and it will show you the actual cache results, thus adding more context to an otherwise meaningless result.